packages/injected/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/injected-wallets",
-  "version": "2.5.0",
+  "version": "2.5.1-alpha.1",
```

Comment:
```suggestion
  "version": "2.6.0-alpha.1",
```
Since this is a pretty big feature change bumping to 2.6

Reply:
Yeah fair call.